src/website/app/pages/ComponentStatus/componentStatus.md

```diff
@@ -17,29 +17,66 @@ Statuses:
-| Grid                                        | [planned]        |
-| Toggle                                      | [planned]        |
-| DataTable                                   |                  |
-| DatePicker                                  |                  |
-| Navigation                                  |                  |
+| Component                                   | Status      |
+|---------------------------------------------|-------------|
+| [Avatar](/components/avatar)                | [available] |
+| [Box](/components/box)                      | [available] |
+| [Button](/components/button)                | [available] |
+| [Card](/components/card)                    | [available] |
+| [Checkbox](/components/checkbox)            | [available] |
+| [Dropdown](/components/dropdown)            | [available] |
+| [Flex](/components/flex)                    | [available] |
+| [Form](/components/form-field)              | [available] |
+| [Icon](/components/icon)                    | [available] |
+| [Link](/components/link)                    | [available] |
+| [Menu](/components/menu)                    | [available] |
+| [Popover](/components/popover)              | [available] |
+| [Radio](/components/radio)                  | [available] |
+| [Select](/components/select) ~ new          | [available] |
+| [StartEnd](/components/start-end)           | [available] |
+| [TextArea](/components/text-area)           | [available] |
+| [TextInput](/components/text-input)         | [available] |
+| [ThemeProvider](/components/theme-provider) | [available] |
+| [Tooltip](/components/tooltip)              | [available] |
+| Grid                                        | [planned]   |
+| Toggle                                      | [planned]   |
+| Table                                       |             |
+| DatePicker                                  |             |
+| Navigation                                  |             |
+| Pagination                                  |             |
+| Tabs                                        |             |
+| Notification                                |             |
+| Text                                        |             |
+| Heading                                     |             |
+| Progress                                    |             |
+| TimePicker                                  |             |
+| Dialog                                      |             |
+| Tile Grid                                   |             |
+| Column Grid                                 |             |
+| List                                        |             |
+| ControlGroup                                |             |
+| StackedLayout                               |             |
+| Tag                                         |             |
+| Badge                                       |             |
+| TypeAhead                                   |             |
```

Comment:
I think Divider is missing from the above group (will do's) & left-nav is missing from the below group (should do's)